spec/inverted-index-test.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+const books = require('./books');
+
+// A test suite to read book data
+describe('Inverted Index Suite', () => {
+  //Create an instance of the Index class
```

Comment:
There should be a space between `//` and `Create...`. Checkout the eslint `spaced-comment` rule (It's a part of AirBnB guidelines)